shared/common-adapters/avatar.native.js

```diff
@@ -3,7 +3,8 @@ import * as shared from './avatar.shared'
 import React, {Component} from 'react'
 import _ from 'lodash'
 import type {Props} from './avatar'
-import {NativeImage, NativeTouchableOpacity, Box} from './index.native'
+import {Box} from './index.native'
+import {TouchableOpacity, Image} from 'react-native'
```

Comment:
common adapters should be the only thing that actually directly require react-native